pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -28,6 +51,7 @@ Create a new NFS partition in your Zpool:
 
 ![Create an NFS partition](images/create-nfs-partition.png){.thumbnail}
 
+#### Your cluster is installed with Public Network or with a private network without a gateway
```

Comment:
Maybe we could reused the naming used in the control panel here?
`OVHcloud Internet Gateway` instead of `gateway`?

Reply:
The field is also named `defaultVrackGateway` on APIv6 side.